### PR TITLE
Add linkcheck to Sphinx instructions

### LINKED ISSUE #455: Add linkcheck instructions to sphinx container documentation
As the title suggests, include the `make linkcheck` option for local linkcheck testing (instead of relying on it only during the PR submission, where it is automatically checked).

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #455. 

### This pull request's proposal
 Changes to be committed:
	modified:   contribute/contribute_documentation.rst
- Add Sphinx linkcheck documentation before submitting for installed and
  docker version
- some cosmetic formatting changes

### Checklist
- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: contribute/contribute_documentation.rst [hunk 1]
```diff
@@ -170,6 +170,27 @@ the files and changes you make to them.
 After making any changes to a *.rst* file, run :code:`make html` as shown above
 and refresh your browser to display your changes.
 
+In order to check hyperlinks that are part of the changed files, it is also recommended
+to run the `linkcheck <https://www.sphinx-doc.org/en/master/usage/builders/index.html>`_
+tool that comes with Sphinx. During PR submission that link check is also runs automatically.
```
Comment (FroggyFlox) [inline contribute/contribute_documentation.rst:175]: Minor typo:
```suggestion
tool that comes with Sphinx. During PR submission that link check is also run automatically.
```

**Review by FroggyFlox — CHANGES_REQUESTED** [review]
Thanks a lot for taking care of this so quickly!
I just noticed a typo, otherwise I'm personally all good with it.

**Review by FroggyFlox — APPROVED** [review]

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks again. Nice find and fix.
@FroggyFlox Thanks for the review: much appreciated.

Merging & publishing directly.

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**